.github/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2020-Present Okta, Inc.
```

Comment:
2021-Present?

Reply:
it was copied from an existing file, so the header _should_ be the same